packages/notifi-react-example/src/NotifiCard/NotifiCard.tsx

```diff
@@ -39,6 +39,12 @@ export const NotifiCard: React.FC = () => {
     },
   };
 
+  const intercomInputSeparators: NotifiInputSeparators = {
```

Comment:
This works! 

In the future, I recommend looking into `pick` util function where you can filter object properties from a main object (in this case, filter from `inputSeparators` to create an `intercomInputSeparators` with only `emailSeparator`): https://stackoverflow.com/questions/35667377/how-to-select-properties-from-an-object-in-javascript

Since this is the React card example, I think that file will change for individual testing purposes as it's okay to not implement a `pick` utils function.